src/data/quizzes/questionBank.ts

```diff
@@ -1,1746 +1,165 @@
-      {
-        id: "staking-2-a",
-        label: "staking-2-a-label",
-        explanation: "staking-2-a-explanation",
-      },
-      {
-        id: "staking-2-b",
-        label: "staking-2-b-label",
-        explanation: "staking-2-b-explanation",
-      },
-      {
-        id: "staking-2-c",
-        label: "staking-2-c-label",
-        explanation: "staking-2-c-explanation",
-      },
-      {
-        id: "staking-2-d",
-        label: "staking-2-d-label",
-        explanation: "staking-2-d-explanation",
-      },
-    ],
-    correctAnswerId: "staking-2-b",
-  },
-  "staking-3": {
-    prompt: "staking-3-prompt",
-    answers: [
-      {
-        id: "staking-3-a",
-        label: "staking-3-a-label",
-        explanation: "staking-3-a-explanation",
-      },
-      {
-        id: "staking-3-b",
-        label: "staking-3-b-label",
-        explanation: "staking-3-b-explanation",
-      },
-      {
-        id: "staking-3-c",
-        label: "staking-3-c-label",
-        explanation: "staking-3-c-explanation",
-      },
-      {
-        id: "staking-3-d",
-        label: "staking-3-d-label",
-        explanation: "staking-3-d-explanation",
-      },
-    ],
-    correctAnswerId: "staking-3-b",
-  },
-  "staking-4": {
-    prompt: "staking-4-prompt",
-    answers: [
-      {
-        id: "staking-4-a",
-        label: "staking-4-a-label",
-        explanation: "staking-4-a-explanation",
-      },
-      {
-        id: "staking-4-b",
-        label: "staking-4-b-label",
-        explanation: "staking-4-b-explanation",
-      },
-      {
-        id: "staking-4-c",
-        label: "staking-4-c-label",
-        explanation: "staking-4-c-explanation",
-      },
-      {
-        id: "staking-4-d",
-        label: "staking-4-d-label",
-        explanation: "staking-4-d-explanation",
-      },
-    ],
-    correctAnswerId: "staking-4-d",
-  },
-  "staking-5": {
-    prompt: "staking-5-prompt",
-    answers: [
-      {
-        id: "staking-5-a",
-        label: "staking-5-a-label",
-        explanation: "staking-5-a-explanation",
-      },
-      {
-        id: "staking-5-b",
-        label: "staking-5-b-label",
-        explanation: "staking-5-b-explanation",
-      },
-      {
-        id: "staking-5-c",
-        label: "staking-5-c-label",
-        explanation: "staking-5-c-explanation",
-      },
-      {
-        id: "staking-5-d",
-        label: "staking-5-d-label",
-        explanation: "staking-5-d-explanation",
-      },
-    ],
-    correctAnswerId: "staking-5-c",
-  },
-  "staking-6": {
-    prompt: "staking-6-prompt",
-    answers: [
-      {
-        id: "staking-6-a",
-        label: "staking-6-a-label",
-        explanation: "staking-6-a-explanation",
-      },
-      {
-        id: "staking-6-b",
-        label: "staking-6-b-label",
-        explanation: "staking-6-b-explanation",
-      },
-      {
-        id: "staking-6-c",
-        label: "staking-6-c-label",
-        explanation: "staking-6-b-explanation",
```

Comment:
And here

<img width="638" alt="Screenshot 2024-12-11 at 22 45 58" src="https://github.com/user-attachments/assets/144950f2-b370-4586-b165-696ffcf5c10c" />